javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,39 @@
+//A future calculator
+const yearOfBirth = 1984;
+let yearFuture = 2027;
```

Comment:
micro advice 
yearFuture and age variables are not changed anywhere. Consequently it's better to use`const`